ecgtools/parsers/cesm.py

```diff
@@ -1,10 +1,59 @@
+import dataclasses
 import pathlib
 
 import cf_xarray  # noqa
 import xarray as xr
 
 from ..core import extract_attr_with_regex
 
+_STREAMS_DICT = {
+    'cam.h0': {'component': 'atm', 'frequency': 'month_1'},
+    'cam.h1': {'component': 'atm', 'frequency': 'day_1'},
+    'cam.h2': {'component': 'atm', 'frequency': 'hour_6'},
+    'cam.h3': {'component': 'atm', 'frequency': 'hour_3'},
+    'cam.h4': {'component': 'atm', 'frequency': 'hour_1'},
+    'cam.h5': {'component': 'atm', 'frequency': 'subhour_3'},
+    'cam.h6': {'component': 'atm', 'frequency': 'day_1'},
+    'cam.h7': {'component': 'atm', 'frequency': 'day_5'},
+    'cam.h8': {'component': 'atm', 'frequency': 'day_10'},
+    'clm2.h0': {'component': 'lnd', 'frequency': 'month_1'},
+    'clm.h1': {'component': 'lnd', 'frequency': 'month_1'},
+    'clm.h2': {'component': 'lnd', 'frequency': 'month_1'},
+    'clm.h3': {'component': 'lnd', 'frequency': 'day_365'},
+    'clm.h4': {'component': 'lnd', 'frequency': 'day_365'},
+    'clm.h5': {'component': 'lnd', 'frequency': 'day_1'},
+    'clm.h6': {'component': 'lnd', 'frequency': 'day_1'},
+    'clm.h7': {'component': 'lnd', 'frequency': 'hour_3'},
+    'clm.h8': {'component': 'lnd', 'frequency': 'day_1'},
+    'mosart.h0': {'component': 'rof', 'frequency': 'month_1'},
+    'mosart.h1': {'component': 'rof', 'frequency': 'day_1'},
+    'mosart.h2': {'component': 'rof', 'frequency': 'hour_6'},
+    'mosart.h3': {'component': 'rof', 'frequency': 'hour_3'},
+    'pop.h': {'component': 'ocn', 'frequency': 'month_1'},
+    'pop.h.nday1': {'component': 'ocn', 'frequency': 'day_1'},
+    'pop.h.nyear1': {'component': 'ocn', 'frequency': 'year_1'},
+    'pop.h.ecosys': {'component': 'ocn', 'frequency': 'month_1'},
+    'pop.h.ecosys.nyear1': {'component': 'ocn', 'frequency': 'year_1'},
+    'cice.h': {'component': 'ice', 'frequency': 'month_1'},
+    'cice.h1': {'component': 'ice', 'frequency': 'day_1'},
+    'cism.h': {'component': 'glc', 'frequency': 'year_1'},
+    'cism.h1': {'component': 'glc', 'frequency': 'month_1'},
+    'ww3.h': {'component': 'wav', 'frequency': 'month_1'},
+}
```

Comment:
@mgrover1, this is the list you gave me earlier today. Is it true that it's missing some streams? If so, could you point me to the documentation page with all the streams? Thanks!

Reply:
Most of them are in there - the only ones that are missing are some the additional 'rof' streams in https://github.com/NCAR/CESM_postprocessing/blob/master/Config/config_timeseries.xml

Reply:
It looks good to start with though!

Reply:
Sounds good... I will use the existing list (the one you gave me) for the time being